vivisect/parsers/pe.py

```diff
@@ -258,8 +260,9 @@ def loadPeIntoWorkspace(vw, pe, filename=None, baseaddr=None):
         if secrva <= entryrva and entryrva < secrvamax:
             mapflags |= e_mem.MM_EXEC
 
-        if not vw.config.viv.parsers.pe.nx and subsys_majver < 6 and mapflags & e_mem.MM_READ:
-            mapflags |= e_mem.MM_EXEC
+        if not nxcompat:
+            if not vw.config.viv.parsers.pe.nx and subsys_majver < 6 and mapflags & e_mem.MM_READ:
```

Comment:
ditto here.
it seems like the maps will only be marked executable if not nxcompat?